scripts/maasutils.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+import click
+import pyrax
+import requests
+
+
+@click.option("--username", required=True)
+@click.option("--api-key", required=True)
+@click.group()
+def cli(api_key, username):
+    pyrax.set_setting("identity_type", "rackspace")
+    pyrax.set_credentials(
+        username,
+        api_key
+    )
+
+
+@click.command()
+def get_token_url():
+    token = pyrax.identity.token
+
+    monitoring_service = next(
+        b for b in pyrax.identity.service_catalog if b["type"] == "rax:monitor"
```

Comment:
How do we ensure that the iterable of pyrax.identity.service_catalog is fine?
Should something special be done if we reach the end of the iterator (raise when exception StopIterator?)